lib/tasks/start_coordinates.rake

```diff
@@ -0,0 +1,14 @@
+desc "Populate start coordinates of stream with data from first measurement"
+task :populate_start_coordinates => [:environment] do
+  Stream.all.each do |stream|
+    first_measurement = stream.measurements.order(time: :asc).first
+    if first_measurement
```

Comment:
The logic here is really similar to the one in the repo, maybe we can remove the duplication?